### DISCUSSION TITLE
Hey!

### DISCUSSION THREAD

**Discussion by luanrramos** [community/community#106530] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello World!

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @luanrramos  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be closed and locked to keep our community nice and tidy 🧹.
